src/components/MessagePane/index.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react"
+
+const MessagePane = ({
+  id,
+  formatDate,
+  closeButton,
+  setCurrentEmailId,
+  setShowCloseButton,
+}) => {
+  const [currentEmailData, setCurrentEmailData] = useState({})
+
+  const closeEmail = () => {
+    setCurrentEmailId("")
```

Comment:
This isn't by default a string. You shouldn't set it to an empty string to blank it out. What was your default value?